### PR TITLE
Add PasswordBox.cs with PasswordBox class

### PR DESCRIPTION
**Pull request by xadaemon**
Adds a password text filed with customizable text mask as a control.

### CODE REVIEW

**Comment by igor-kostromin** [comment]
Thanks for contributing ! I'll take a look over next few days )

File: ConsoleFramework/Controls/PasswordBox.cs [hunk 1]
```diff
@@ -0,0 +1,291 @@
+﻿using System;
+using System.Collections.Generic;
+using ConsoleFramework.Core;
+using ConsoleFramework.Events;
+using ConsoleFramework.Native;
+using ConsoleFramework.Rendering;
+
+namespace ConsoleFramework.Controls
+{
+    public class PasswordBox : Control
```
Comment (igor-kostromin) [inline ConsoleFramework/Controls/PasswordBox.cs:10]: I think it will be good to try to inherit PasswordBox from TextBox and avoid some copypaste code

Comment (xadaemon) [reply ConsoleFramework/Controls/PasswordBox.cs:10]: It was old code I had written a long time ago I will fix it and open a new request

**Comment by xadaemon** [comment]
Closing will open a new one as soon as I fix all the problems

**Pull request closed without merging**